### PR TITLE
Add -12 change log.

### PR DESCRIPTION
**Pull request by chris-wood**
cc @cjpatton, @ekr, @davidben, @sftcd

### CODE REVIEW

File: draft-ietf-tls-esni.md [hunk 1]
```diff
@@ -1717,6 +1717,14 @@ important ideas and contributions.
...
 Issue and pull request numbers are listed with a leading octothorp.
 
+## Since draft-ietf-tls-esni-11
+
+- Move ClientHello padding to the encoding (#443)
+
+- Relax OuterExtensions checks for alignment with RFC8446 (#467)
+
+- Editorial improvements (#468, #465, #462, #461)
+
```
Comment (cjpatton) [inline draft-ietf-tls-esni.md:1727]: I think we should call out #464 as a bugfix. ECH-11 isn't implementable.

Comment (chris-wood) [reply draft-ietf-tls-esni.md:1729]: Well, not really. It just broke the convention.

Comment (chris-wood) [reply draft-ietf-tls-esni.md:1729]: Fixed anyway.

**Comment by chris-wood** [comment]
Happy to park this if we want to land a change for #466, but I don't think that's absolutely needed for a new version.

File: draft-ietf-tls-esni.md (hunk 1 above)
Comment (davidben) [reply draft-ietf-tls-esni.md:1729]: Well, it was a convention break along with an incompatibility with ECH-10. So I guess it's implementable, but you don't want to implement it the way it was written. :-)

**Review by davidben — APPROVED** [review]
LGTM, though I think a fix for #466 would be prudent since it's currently ambiguous.

**Review by cjpatton — APPROVED** [review]

**Pull request merged**